mappings/net/minecraft/world/gen/DensityFunctions.mapping

```diff
@@ -0,0 +1,445 @@
+CLASS net/minecraft/unmapped/C_mdkmcezi net/minecraft/world/gen/DensityFunctions
+	FIELD f_ezohebie DIRECT_CODEC Lcom/mojang/serialization/Codec;
+	FIELD f_ikpluaeq MAX_REASONABLE_NOISE_VALUE D
+	FIELD f_ofybcvpy CODEC Lcom/mojang/serialization/Codec;
+	FIELD f_zreaxxak NOISE_VALUE_CODEC Lcom/mojang/serialization/Codec;
+	METHOD m_ajxltzce blendOffset ()Lnet/minecraft/unmapped/C_kzwapyud;
+	METHOD m_akixioyj blendDensity (Lnet/minecraft/unmapped/C_kzwapyud;)Lnet/minecraft/unmapped/C_kzwapyud;
+		ARG 0 input
+	METHOD m_bavhhzle terrainShaperSpline (Lnet/minecraft/unmapped/C_kzwapyud;Lnet/minecraft/unmapped/C_kzwapyud;Lnet/minecraft/unmapped/C_kzwapyud;Lnet/minecraft/unmapped/C_mdkmcezi$C_uejbabyt$C_zlreguhf;DD)Lnet/minecraft/unmapped/C_kzwapyud;
+	METHOD m_bnrcnkgf shift (Lnet/minecraft/unmapped/C_cjzoxshv;)Lnet/minecraft/unmapped/C_kzwapyud;
+	METHOD m_diiubzxc shiftedNoise2d (Lnet/minecraft/unmapped/C_kzwapyud;Lnet/minecraft/unmapped/C_kzwapyud;DLnet/minecraft/unmapped/C_cjzoxshv;)Lnet/minecraft/unmapped/C_kzwapyud;
+	METHOD m_ewjecyar doubleFunctionArgumentCodec (Ljava/util/function/BiFunction;Ljava/util/function/Function;Ljava/util/function/Function;)Lcom/mojang/serialization/Codec;
+	METHOD m_fezycrnr shiftB (Lnet/minecraft/unmapped/C_cjzoxshv;)Lnet/minecraft/unmapped/C_kzwapyud;
+	METHOD m_gswmyjbd clampedGradientY (IIDD)Lnet/minecraft/unmapped/C_kzwapyud;
+		ARG 0 fromY
+		ARG 1 toY
+		ARG 2 fromValue
+		ARG 4 toValue
+	METHOD m_harrkprv zero ()Lnet/minecraft/unmapped/C_kzwapyud;
+	METHOD m_hpogshhq min (Lnet/minecraft/unmapped/C_kzwapyud;Lnet/minecraft/unmapped/C_kzwapyud;)Lnet/minecraft/unmapped/C_kzwapyud;
+		ARG 0 function1
+		ARG 1 function2
+	METHOD m_icmhodbz mappedNoise (Lnet/minecraft/unmapped/C_cjzoxshv;DD)Lnet/minecraft/unmapped/C_kzwapyud;
+		ARG 0 noise
+	METHOD m_irznzryl singleArgumentCodec (Lcom/mojang/serialization/Codec;Ljava/util/function/Function;Ljava/util/function/Function;)Lcom/mojang/serialization/Codec;
+	METHOD m_jtpnyujf lerp (Lnet/minecraft/unmapped/C_kzwapyud;Lnet/minecraft/unmapped/C_kzwapyud;Lnet/minecraft/unmapped/C_kzwapyud;)Lnet/minecraft/unmapped/C_kzwapyud;
+		ARG 0 function
+		ARG 1 value1
+		ARG 2 value2
+	METHOD m_khwsifkt shiftA (Lnet/minecraft/unmapped/C_cjzoxshv;)Lnet/minecraft/unmapped/C_kzwapyud;
```

Comment:
shiftA?

Reply:
One of 3 shift functions it seems, one shifts `x, 0, z`, one shifts `z, 0, x` the other shifts `x, y, z`. I'm going to change the names to their appropriate coordinate